tests/Unit/PointwiseFunctions/Hydro/TestHelpers.hpp

```diff
@@ -0,0 +1,64 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <random>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace hydro {
+/// \ingroup TestingFrameworkGroup
+/// \brief Make random hydro variables which correct physical behavior,
+/// e.g. Lorentz factor will be greater or equal than one.
+namespace TestHelpers {
+template <typename DataType>
+Scalar<DataType> random_density(const gsl::not_null<std::mt19937*> generator,
+                                const DataType& used_for_size) noexcept;
+
+template <typename DataType>
+Scalar<DataType> random_lorentz_factor(
+    const gsl::not_null<std::mt19937*> generator,
+    const DataType& used_for_size) noexcept;
+
+template <typename DataType, size_t Dim>
+tnsr::I<DataType, Dim> random_velocity(
+    const gsl::not_null<std::mt19937*> generator,
+    const Scalar<DataType>& lorentz_factor,
+    const tnsr::ii<DataType, Dim>& spatial_metric) noexcept;
+
+template <typename DataType>
+Scalar<DataType> random_temperature(
+    const gsl::not_null<std::mt19937*> generator,
+    const DataType& used_for_size) noexcept;
+
+template <typename DataType>
+Scalar<DataType> random_specific_internal_energy(
+    const gsl::not_null<std::mt19937*> generator,
+    const DataType& used_for_size) noexcept;
+
+template <typename DataType>
+tnsr::I<DataType, 3> random_magnetic_field(
+    const gsl::not_null<std::mt19937*> generator,
+    const Scalar<DataType>& pressure,
+    const tnsr::ii<DataType, 3>& spatial_metric) noexcept;
+
+template <typename DataType>
+Scalar<DataType> random_divergence_cleaning_field(
+    const gsl::not_null<std::mt19937*> generator,
+    const DataType& used_for_size) noexcept;
+
+// Needed in some tests, so we put it here for convenience
+template <size_t Dim, typename DataType>
+tnsr::ii<DataType, Dim> random_spatial_metric(
```

Comment:
This one (and maybe `random_lorentz_factor` and `random_velocity`) is more general than just hydro.  We should move it to somewhere in GR, but that can be put off until someone wants it if you want.

Reply:
I agree. Maybe the tests in `tests/Unit/PointwiseFunctions/GeneralRelativity/TestHelpers.hpp` can be reworked similarly in the future.